runtime/src/enclave_rpc/demux.rs

```diff
@@ -15,13 +15,13 @@ use super::{
 use crate::{common::time::insecure_posix_system_time, identity::Identity};
 
 /// Maximum concurrent EnclaveRPC sessions.
-const MAX_CONCURRENT_SESSIONS: usize = 100;
+const MAX_CONCURRENT_SESSIONS: usize = 1024;
 /// Sessions without any processed frame for more than STALE_SESSION_TIMEOUT_SECS seconds
 /// can be purged.
-const STALE_SESSION_TIMEOUT_SECS: u64 = 60;
+const STALE_SESSION_TIMEOUT_SECS: u64 = 5;
```

Comment:
Big changes. Should be tested thoroughly on testnet.

Reply:
Will revert this part and add support for per peer limits instead.